flint.ui/src/components/Datepicker/DatepickerPoint.vue

```diff
@@ -79,9 +81,11 @@ export default {
         selectedEndDate.value = val
         console.log(selectedEndDate.value)
         if (date_diff.value < 0) {
-          notification.error({
+          useToast({
+            type: 'error',
+            title: 'Error',
```

Comment:
Here we can change from "Error" to "Invalid End Date".

Reply:
Done